.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ repos:
   - id: check-merge-conflict
 
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.7.3
+  rev: v0.8.0
```

Comment:
Is there a way that we can set this so that it looks for the latest version, to avoid having to manually update it like this?

Reply:
I created a separate PR for this https://github.com/fitbenchmarking/fitbenchmarking/pull/1391. I tested it on a private repo last week and it works. Hopefully it will get merged soon and in the future it will automatically be updated by GitHub Actions.

Reply:
Okay brilliant, thanks @RabiyaF!